src/vivarium_csu_ltbi/components/names.py

```diff
@@ -0,0 +1,31 @@
+# disease name
+TUBERCULOSIS_AND_HIV = 'tuberculosis_and_hiv'
+
+
+#  state names
+ACTIVETB_POSITIVE_HIV = 'activetb_positive_hiv'
+ACTIVETB_SUSCEPTIBLE_HIV = 'activetb_susceptible_hiv'
+LTBI_SUSCEPTIBLE_HIV = 'ltbi_susceptible_hiv'
+LTBI_POSITIVE_HIV = 'ltbi_positive_hiv'
+PROTECTED_TB_SUSCEPTIBLE_HIV = 'protected_tb_susceptible_hiv'
+PROTECTED_TB_POSITIVE_HIV = 'protected_tb_positive_hiv'
+SUSCEPTIBLE_TB_POSITIVE_HIV = 'susceptible_tb_positive_hiv'
+SUSCEPTIBLE_TB_SUSCEPTIBLE_HIV = 'susceptible_tb_susceptible_hiv'
+
+
+# transition names (from, to)
+ACTIVETB_SUSCEPTIBLE_HIV_TO_ACTIVETB_POSITIVE_HIV = 'activetb_susceptible_hiv_to_activetb_positive_hiv'
+ACTIVETB_SUSCEPTIBLE_HIV_TO_SUSCEPTIBLE_TB_SUSCEPTIBLE_HIV = \
```

Comment:
accidental line break here

Reply:
It is actually too long given the super verbose names